pkg/hostagent/requirements.go

```diff
@@ -137,7 +137,26 @@ A possible workaround is to run "lima-guestagent install-systemd" in the guest.
 
 func (a *HostAgent) optionalRequirements() []requirement {
 	req := make([]requirement, 0)
-	if *a.y.Containerd.System || *a.y.Containerd.User {
+	if *a.y.Containerd.System {
+		req = append(req,
+			requirement{
+				description: "systemd or alpine must be available",
+				fatal:       true,
+				script: `#!/bin/bash
+set -eux -o pipefail
+if ! command -v systemctl 2>&1 >/dev/null -a ! test -f /etc/alpine-release; then
```

Comment:
This doesn't work for me; did you mean:

```suggestion
if ! command -v systemctl 2>&1 >/dev/null && ! test -f /etc/alpine-release; then
```